soc/nordic/soc.yml

```diff
@@ -0,0 +1,9 @@
+family:
+  - name: nordic_nrf
+    series:
+      - name: nrf71
```

Comment:
because of build systems in future, this should match the Kconfig name so both `nrf71` or both `nrf71x` - do not make any changes here, just pinging @carlescufi @tejlmand because this is going to become a big problem